Flaky ITRealtimeIndexTaskTest

```[ERROR] Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1,419.876 s <<< FAILURE! - in TestSuite
[ERROR] testRealtimeIndexTask(org.apache.druid.tests.indexer.ITRealtimeIndexTaskTest)  Time elapsed: 182.638 s  <<< FAILURE!
java.lang.RuntimeException: java.util.concurrent.ExecutionException: org.jboss.netty.channel.ChannelException: Faulty channel in resource pool
	at org.apache.druid.tests.indexer.ITRealtimeIndexTaskTest.postEvents(ITRealtimeIndexTaskTest.java:119)
	at org.apache.druid.tests.indexer.ITRealtimeIndexTaskTest.testRealtimeIndexTask(ITRealtimeIndexTaskTest.java:57)
Caused by: java.util.concurrent.ExecutionException: org.jboss.netty.channel.ChannelException: Faulty channel in resource pool
	at org.apache.druid.tests.indexer.ITRealtimeIndexTaskTest.postEvents(ITRealtimeIndexTaskTest.java:119)
	at org.apache.druid.tests.indexer.ITRealtimeIndexTaskTest.testRealtimeIndexTask(ITRealtimeIndexTaskTest.java:57)
Caused by: org.jboss.netty.channel.ChannelException: Faulty channel in resource pool
	at org.apache.druid.tests.indexer.ITRealtimeIndexTaskTest.postEvents(ITRealtimeIndexTaskTest.java:119)
	at org.apache.druid.tests.indexer.ITRealtimeIndexTaskTest.testRealtimeIndexTask(ITRealtimeIndexTaskTest.java:57)
Caused by: org.jboss.netty.channel.ChannelException: Failed to handshake with host[https://127.0.0.1:8101]
Caused by: java.nio.channels.ClosedChannelException
```

--- a/integration-tests/src/test/java/org/apache/druid/tests/indexer/AbstractITRealtimeIndexTaskTest.java
+++ b/integration-tests/src/test/java/org/apache/druid/tests/indexer/AbstractITRealtimeIndexTaskTest.java
@@ -97,6 +97,8 @@ public abstract class AbstractITRealtimeIndexTaskTest extends AbstractIndexerTes
       LOG.info("indexerSpec: [%s]\n", task);
       taskID = indexer.submitTask(task);
 
+
+      // sleep for a while to let peons finish starting up
       TimeUnit.SECONDS.sleep(5);
 
       // this posts 22 events, one every 4 seconds

--- a/integration-tests/src/test/java/org/apache/druid/tests/indexer/AbstractITRealtimeIndexTaskTest.java
+++ b/integration-tests/src/test/java/org/apache/druid/tests/indexer/AbstractITRealtimeIndexTaskTest.java
@@ -97,6 +97,8 @@ public abstract class AbstractITRealtimeIndexTaskTest extends AbstractIndexerTes
       LOG.info("indexerSpec: [%s]\n", task);
       taskID = indexer.submitTask(task);
 
+      TimeUnit.SECONDS.sleep(5);
+
       // this posts 22 events, one every 4 seconds
       // each event contains the current time as its timestamp except
       //   the timestamp for the 14th event is early enough that the event should be ignored

